applications/moflow/Dockerfile

```diff
@@ -0,0 +1,82 @@
+# Base image specification
+ARG FROM_IMAGE=ubuntu:24.04
+FROM ${FROM_IMAGE} as builder
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Proxy configuration
+ARG http_proxy
+ENV http_proxy=${http_proxy}
+
+ARG https_proxy
+ENV https_proxy=${https_proxy}
+
+ARG no_proxy
+ENV no_proxy=${no_proxy}
+
+# Install necessary build tools and clean up
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    git build-essential wget vim ca-certificates autoconf automake make numactl unzip && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get autoremove -y && \
+    apt-get clean
+
+# Create a user and group for running the service
+ENV SERVICE_NAME="moflow-base-service"
+RUN groupadd --gid 1001 $SERVICE_NAME && \
+    useradd -m -g $SERVICE_NAME --shell /bin/false --uid 1001 $SERVICE_NAME
+
+WORKDIR /app
+RUN chown -R $SERVICE_NAME:$SERVICE_NAME /app
+USER $SERVICE_NAME
+
+# Install Miniforge (Conda)
+RUN wget --no-check-certificate -O Miniforge3.sh "https://github.com/conda-forge/miniforge/releases/download/24.3.0-0/Miniforge3-$(uname)-$(uname -m).sh"
+RUN bash Miniforge3.sh -b -p "${HOME}/conda" && rm Miniforge3.sh
+
+# Clone MoFlow repository and apply patch
+RUN git clone --recursive https://github.com/calvin-zcx/moflow.git moflow
+WORKDIR /app/moflow
+RUN git checkout 3611c637260272b3d34a298f221623cb59e01091
+
+RUN wget https://github.com/IntelLabs/Open-Omics-Acceleration-Framework/releases/download/3.0/Source_code_with_submodules.tar.gz 
+RUN tar -xzf Source_code_with_submodules.tar.gz
+
+WORKDIR /app/moflow
```

Comment:
This is redundant. We are already in /app/moflow

Reply:
I have removed the WORKDIR line from the Dockerfile.